apps/web/styles/globals.css

```
@@ -58,36 +70,37 @@ html.dark {
 	}
 
 	:root {
-		--background: 0 0% 100%;
-		--foreground: 222.2 84% 4.9%;
 
-		--muted: 210 40% 96.1%;
-		--muted-foreground: 215.4 16.3% 46.9%;
+		--background: 240 5% 96%;
```

Stylelint (17.4.0)

[error] Unexpected empty line before custom property (custom-property-empty-line-before)

(custom-property-empty-line-before)